pytest_catchlog.py

```diff
@@ -154,6 +175,73 @@ def pytest_runtest_teardown(self, item):
         with self._runtest_for(item, 'teardown'):
             yield
 
+    def configure_console_handler(self, verbosity):
+        # Prepare the handled_levels dictionary
+        log_levels = []
+        handled_levels = {}
+        available_levels = set([
+            #logging.CRITICAL,  # This will be the default console level if no -v is passed
+            logging.ERROR,
+            logging.WARNING,
+            logging.INFO,
+            logging.DEBUG,
+            logging.NOTSET
+        ])
+        for level in get_option_ini(self.config, 'log_extra_levels'):
+            try:
+                level_num = int(level)
+            except ValueError:
+                level_num = logging.getLevelName(level)
```

Comment:
Don't know if that's publicly documented but that behavior exists in py2 and py3....